apps/src/studio/studio.js

```diff
@@ -5916,6 +5974,7 @@ Studio.moveSingle = function (opts) {
   }
 
   if (skin.gridAlignedMovement) {
+    sprite.setActivity(constants.BEHAVIOR_GRID_ALIGNED);
```

Comment:
This seems good to me.  It's actually weirder that I don't see a corresponding `setActivity` call in the else case.